packages/spanner/src/database.js

```diff
@@ -713,45 +713,44 @@ Database.prototype.runStream = function(query, options) {
  * });
  *
  * //-
- * // If the callback is omitted, we'll return a Promise.
+ * // To use promises, simply return a Promise.
  * //-
- * database.runTransaction()
- *   .then(function(data) {
- *     var transaction = data[0];
- *
- *     // Run a transactional query.
- *     return transaction.run('SELECT * FROM Singers')
- *       .then(function() {
- *         // Queue a mutation (note there is no callback passed to `insert`).
- *         transaction.insert('Singers', {
- *           SingerId: 'Id3b',
- *           Name: 'Joe West'
- *         });
- *
- *         // Commit the transaction.
- *         return transaction.commit();
+ * database.runTransaction(function(err, transaction) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   // Run a transactional query.
+ *   return transaction.run('SELECT * FROM Singers')
```

Comment:
Generally I dislike style discussions in code reviews, but I had to re-read this a couple times because of the odd nesting and inconsistently aligned `.then`s.

Can this be changed to:
```javascript
return transaction.run('SELECT * FROM Singers').then(function() {
```

If that exceeds 80 characters, change `transaction` to `txn`. It will still end up more readable.